### ISSUE TITLE
[Playlist Tags] Tags Disappering

### ISSUE THREAD

**Issue by Amiru2007** [Bergbok/Spicetify-Creations#5]
### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.2.35.663.gb699649e
- Spicetify version: v2.36.8
```

### 📝 Description

I've lost my tags a few days ago and added the tags again. So, I went to the **tags** custom app page then to **All Tags** Tab. Nothing was wrong their. Then I clicked one of the tags to view the playlist that tag contains but I was redirected to the app's **search** tab and displayed no playlist for that tag. So I checked the **All Tags** page again to click it back but there was no such tag I clicked before. To ensure I've clicked another tag and similarly search said there isn't any playlist. So again I checked the **All Tags** Page and that tag also got disappeared. Also **All tagged Playlists** page is also empty and doesn't show any playlists.

#### No Tagged Playlists
![Spotify Free 4_25_2024 11_07_55 PM](https://github.com/Bergbok/Spicetify-Creations/assets/79262786/5c189671-b1cc-4f34-baeb-37ed99d8339a)
#### No playlists shown on search
![Spotify Free 4_25_2024 11_07_51 PM](https://github.com/Bergbok/Spicetify-Creations/assets/79262786/bb7a0085-b90a-401f-ba20-7be8a817af84)

**Comment by Bergbok** [comment]
Sorry to hear you lost your tags! Did you happen to run a Spotify installer before it happened? 
That's the only way my tags have gotten cleared. 

I recommend keeping backups in case that happens again, you can export your tags in settings.

The playlist cards stopped rendering after an update recently, not sure if it was a Spotify or Spicetify update that broke it. 
Probably Spotify. I already opened a [issue ](https://github.com/harbassan/spicetify-apps/issues/87) on the repo that I got the code from.

To confirm whether or not your issue is related to that please:
1. Enter some tags that should match a playlist
2. Click the play button and let me know whether playing still works or not

Also, just to confirm - your tags are still rendering on playlist and the All Tags pages, right?

**Comment by Amiru2007** [comment]
I lost them in the Spotify auto update ran recently. But I added them each playlist again.

The **play button works** fine and there's no issues in playlist pages or on **All Tags** page.

_Don't worry about the sidebar custom app icon not showing, it is an issue in the theme I installed._

**Comment by Bergbok** [comment]
Ahh, typically auto-updates don't clear them but I guess the one to V1.2.35 must've.

I see you retracted this statement: **"Also, In All Tags page when I click a tag it is searched in the search bar similar showing no playlists. and when I look all tags page again the tag has been deleted automatically."** - can I assume this doesn't happen anymore?
The only ways I coded to remove tags are when you right click on them or by using the mass removal settings.

If the play button still allows you to play your tagged playlists all is good. The invisible playlists should be fixed soon™.

_Okay cool, was gonna ask about it - what theme are you using out of curiosity?_

**Comment by Amiru2007** [comment]
Sorry, my mistake tags won't delete automatically. The tags I thought disappeared were sent to the beginning of the tag list. 

I'm using the [Bloom](https://github.com/nimsandu/spicetify-bloom) theme.
(I've changed some code in the user.css of the theme I've installed manually and changed the icons in my opinion, in original one only the search and home icons do appear, and custom app icons don't appear at all.) I haven't updated the theme in a while.
